src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.controller;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.entity.Reservation;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private List<Reservation> reservations = new ArrayList<>();
+    private AtomicLong index = new AtomicLong(1);
+
+    @GetMapping()
+    public List<Reservation> getReservations(
```

Comment:
```suggestion
    @GetMapping
    public List<Reservation> getReservations(
```
아무런 path 명시를 하지 않을거라면, `@GetMapping` 으로 선언하실수도 있습니다ㅎㅎ